zone/exp.cpp

```diff
@@ -801,6 +801,17 @@ void Client::SetEXP(uint64 set_exp, uint64 set_aaxp, bool isrezzexp) {
 		}
 	}
 
+    if (m_pp.exp != set_exp) {
+        const auto xp_value = set_exp - m_pp.exp;
+        const auto export_string = fmt::format("{}",xp_value);
+        parse->EventPlayer(EVENT_XP_GAIN, this,export_string, xp_value);
+    }
+    if (m_pp.expAA != set_aaxp) {
+        const auto aaxp_value = set_aaxp - m_pp.expAA;
+        const auto export_string = fmt::format("{}",aaxp_value);
+        parse->EventPlayer(EVENT_AAXP_GAIN, this, export_string, aaxp_value);
+    }
+
```

Comment:
Don't think you **need** to send it in `data` and `extradata` here, just one or the other is fine.

Perl doesn't really care what you send it, so you can just send `data` and leave `extradata` as `0`.

Reply:
Updated